cpp/oneapi/dal/backend/primitives/voting/test/uniform_dpc.cpp

```diff
@@ -0,0 +1,127 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <algorithm>
+#include <cmath>
+#include <limits>
+#include <numeric>
+#include <vector>
+
+#include "oneapi/dal/backend/primitives/common.hpp"
+#include "oneapi/dal/backend/primitives/voting.hpp"
+
+#include "oneapi/dal/test/engine/fixtures.hpp"
+#include "oneapi/dal/test/engine/math.hpp"
+
+namespace oneapi::dal::backend::primitives::test {
+
+namespace te = dal::test::engine;
+namespace de = dal::detail;
+namespace la = te::linalg;
+
+class uniform_test : public te::policy_fixture {
+public:
+    using cls_t = std::int32_t;
+
+    auto compute_results(const ndview<cls_t, 2>& responses) {
+        const auto r = responses.get_dimension(0);
+        const auto k = responses.get_dimension(1);
+        auto tmp = ndarray<cls_t, 2>::empty(this->get_queue(), { r, k });
+        auto res = ndarray<cls_t, 1>::empty(this->get_queue(), { r });
+        copy_by_value(this->get_queue(), tmp, responses).wait_and_throw();
+        for (int j = 0; j < r; ++j) {
```

Comment:
int -> std::int32_t

Reply:
It is test. There is no need to be so strict with types here. I.e.: https://github.com/oneapi-src/oneDAL/blob/fce295e687ae23acb7602a6ef8c5aa248ad71a05/cpp/oneapi/dal/backend/primitives/selection/test/row_partitioning_dpc.cpp#L59

Reply:
Fixed